locales/sv-SE.json

```diff
@@ -38,8 +38,8 @@
         "1": "Håll <RELATIVE_DIRECTION> till <STREET_NAMES>.",
         "2": "Håll <RELATIVE_DIRECTION> till <BEGIN_STREET_NAMES>. Fortsätt på <STREET_NAMES>.",
         "3": "Håll <RELATIVE_DIRECTION> för att stanna kvar på <STREET_NAMES>.",
-        "4": "Bear <RELATIVE_DIRECTION> at <JUNCTION_NAME>.",
-        "5": "Bear <RELATIVE_DIRECTION> toward <TOWARD_SIGN>."
+        "4": "Ta <RELATIVE_DIRECTION> vid <JUNCTION_NAME>.",
+        "5": "Ta <RELATIVE_DIRECTION> mot <TOWARD_SIGN>."
```

Comment:
should the verbs really be different. to me it looks like it says:

"keep RELATIVE_DIRECTION..." in the old translation AND
"take a RELATIVE_DIRECTION..." in the new translation shouldn't we try to be consistent or do the different prepositions at the end of the phrase change the verb that sounds right with them?